metric/metricexport/metric.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metricexport
+
+import (
+	"time"
+
+	"go.opencensus.io/resource"
+)
+
+// LabelValue represents the value of a label. A missing value (nil) is distinct
+// from an empty string value.
+type LabelValue struct {
+	Value   string
+	Present bool
+}
+
+// NewLabelValue creates a new non-nil LabelValue that represents the given string.
+func NewLabelValue(val string) LabelValue {
```

Comment:
How do people get access to a nil LabelValue? Should we create a constant NilLabelValue?

Reply:
They can just use the zero value: LabelValue{}

Reply:
I will document this.